packages/compass-components/.eslintrc.js

```diff
@@ -5,4 +5,7 @@ module.exports = {
     tsconfigRootDir: __dirname,
     project: ['./tsconfig-lint.json'],
   },
+  rules: {
+    '@mongodb-js/compass/no-leafygreen-outside-compass-components': 'off',
```

Comment:
If you look at the rule implementation, it checks in which context it's running and skips check if it's compass components (otherwise our ci would be failing for quite awhile already). No harm in disabling it here, but this might suggest that you have something misconfigured on your machine

Reply:
Hmm. I don't know how that would be. I don't have much configured in vscode.